convert2rhel/cert.py

```diff
@@ -79,3 +122,17 @@ def remove(self):
                 loggerinst.info("No RHSM certificates found to be removed.")
             else:
                 loggerinst.error("OSError({0}): {1}".format(err.errno, err.strerror))
+
+
+def _get_cert(cert_dir):
+    """Return name of certificate and this directory."""
+    if not os.access(cert_dir, os.R_OK | os.X_OK):
+        loggerinst.critical("Error: Could not access %s." % cert_dir)
+    pem_filename = None
+    for filename in os.listdir(cert_dir):
+        if filename.endswith(".pem"):
+            pem_filename = filename
+            break
```

Comment:
Could there ever be more than one pem file?

Reply:
No, at least not under any circumstance I can think of now. If we for any reason need to add more than one .pem file to a dir, we'll need to change this.